### DISCUSSION TITLE
@formily/element-使用arrayTable如何获取选中行数据

### DISCUSSION THREAD

**Discussion by BlueYeah** [alibaba/formily#2235] [Q&A 2.x] [answered]
formily版本： v2
@formily/element

![image](https://user-images.githubusercontent.com/18047981/134662073-568cab63-ff33-419a-8df6-359244549774.png)

需求:
1.点击编辑按钮能获取选中行的数据进行业务编辑。
2.获取的数据如何传到弹窗组件内。

麻烦针对两点给一下思路。 看了好久文档实在没头绪。
辛苦了。

https://codesandbox.io/s/divine-star-rrqo6?file=/src/component/dialog.js

**Comment by muuyao** [comment, marked as answer]
可以通过 useForm 获取 form 实例
https://codesandbox.io/s/bold-zeh-h3dl0?file=/src/component/dialog.js

**Reply by BlueYeah** [reply]
谢谢
